add hall to site angular model

kds1660 opened the pull request:
for now without api requests

kds1660 committed:
Merge remote-tracking branch 'refs/remotes/Geekhub-2014-JS/dev' into dev

kds1660 committed:
add hall to theatre

1. hall at route http://localhost/#/uk/hall
2. in hall controller you can set hall html (for now kulyk or
filarmonia), for api in future select hall map in variable
3. by clicking at sits - add it to card
4. todo - add cost from place-price class? corresponding with dt value
for class in hall header
5. change css fronend for cart (pm added this task for frontend team)
6. bad issue - scope.apply for hall onclick, maibe add ng-click to hall
to add watcher, but for now 'hall'.html work without angular (if it
generated in backend in future)

kds1660 committed:
fix

Varkh commented on src/js/Controllers/hallCtrl.js:
```diff
@@ -0,0 +1,14 @@
+        function ($scope,hallService,cartService) {
+           // hallService.setHall('Filarmonia');
+            hallService.setHall('Kulyka');
+        $scope.getUrl=function () {
```
remove

Varkh commented on src/js/Services/services.js:
```diff
@@ -100,4 +100,40 @@ theatreServices
     })
+
+    .factory('hallService', function() {
+        var hall='';
```
think about getting list of halls from back-end

kds1660 replied:
for now get from backend const halls and add in object
 var hallConst = {
                "Черкаська Філармонія": "Filarmonia",
                "Будинок культури ім. Кулика": "Kulyka",
                "Черкаський Театр": "",
                "Кінотеатр 'Салют'": "",
                "Черкаський міський Палац молоді": "",
                "Центр дитячої та юнацької творчості": ""
            };

kds1660 replied:
for now dont have api, added from const, backend said, that they
will provide for a venue resource, but how i will convert names into html filename, if i receive halls...

Varkh commented on src/js/Services/services.js:
```diff
@@ -100,4 +100,40 @@ theatreServices
+            return order;
+        },
+        delPlaceFromCart: function (index) {
+            console.log(order[index],index)
```
no `console.log`

kds1660 committed:
remove unnecessary code

kds1660 committed:
Halls integration

1) change hall,css and bokking aside (frontend changes)
2) from 1 - change click event function for new html structure
3) in single perfomanse controller add function for button buy tickets
and add service for store selected perfomance
4) in perfomance html add hall name for performance events and button
buy tickets (todo change button view)

kds1660 committed:
hall directive changes

Varkh commented on app/views/Performance/performance.html:
```diff
@@ -23,6 +23,12 @@ <h2 class="page-title">{{performance.title}}</h2>
+                    <span>{{performanceEvents[$index].venue}}</span>
+                    <div class="block-description">
+                        <a ng-click="selectPerfomance($index)">
+                        <span >Придбати квиток</span>
```
As I understand, titles should be moved to file. As there are few languages. ex. `{{perfomance.performance.title}}`

kds1660 replied:
this is strange translation at site, menu-ok, other -50/50. I think about it, i will try translate today, but venue from backend - ukrainian always

Varkh commented on src/js/Controllers/SinglePerfCtrl.js:
```diff
@@ -2,8 +2,8 @@
-    .controller('SinglePerformanceCtrl', ['$scope', '$q', 'apiGet', '$stateParams', 'dateConvert', '$rootScope',
-        function ($scope, $q, apiGet, $stateParams, dateConvert, $rootScope) {
+    .controller('SinglePerformanceCtrl', ['$scope', '$q', 'apiGet', '$stateParams', 'dateConvert', '$rootScope','$state','perfomanceService',
+        function ($scope, $q, apiGet, $stateParams, dateConvert, $rootScope,$state,perfomanceService) {
```
space ` ` before parameter

kds1660 replied:
added spaces

Varkh commented on src/js/Directives/HallDirective.js:
```diff
@@ -0,0 +1,29 @@
+                        }
+                        scope.addToCart(sitsData)
+                    }
+
```
remove unneeded new lines

kds1660 replied:
removed

Varkh commented:
Update changed from DEV, as Merging is blocked
@kds1660

kds1660 committed:
change button view for buy tickets

kds1660 committed:
changes from requested changes

kds1660 committed:
changes filenames/controllre names to theatre naming rules, add 7th hall from back-end

kds1660 committed:
changes filenames/controllre names to theatre naming rules

kds1660 committed:
add translation

1. move legend from halls into separate file with translate
2. trunslate buy button and bookingAside

Varkh commented:
@kds1660  Please, update from dev one more time

kds1660 committed:
change
